src/gr11review/part1/Review7.java

```diff
@@ -1 +1,44 @@
 package gr11review.part1;
+
+import java.util.Scanner;
+
+public class Review7 {
+
+    public static void main(String[] args) {
+
+
+
+        Scanner scanner = new Scanner(System.in);
+
+        String theSentence = scanner.nextLine();
+
+        int CharacterCount = 0;
+        int SpaceCount = 0;
+        int aCount = 0;
+
+        for(int i = 0; i < theSentence.length(); i++){
+                CharacterCount++;
+        }
+        for (int i = 0; i < theSentence.length(); i++){
+            if(theSentence.charAt(i) == ' '){
+                SpaceCount++;
+            }
+        }
+        for (int i = 0; i < theSentence.length(); i++){
+            if(theSentence.charAt(i) == 'a'){
+                aCount++;
+            }
+        }
+
+
+        System.out.println("There are "+CharacterCount+ " characters in the sentence.");
+        System.out.println("There are "+SpaceCount+ " spaces in the sentence.");
+        System.out.println("There are "+aCount+ " letter a in the sentence.");
+        for (int i = 0; i < theSentence.length(); i= i+2){
+            System.out.print(theSentence.charAt(i));
+        }
```

Comment:
It looks like you misinterpreted the the last requirement See https://github.com/SACHSTech/native-java-and-gr-11-review-assignment-amaan2106#review7java---string-manipulation

`Taking the odd numbered characters in the sentence produces the following string “---------“`  that is, a `-` for every odd character.  

https://github.com/SACHSTech/native-java-and-gr-11-review-assignment-amaan2106/runs/2623634154#step:3:537